core/src/main/java/org/apache/carbondata/core/constants/CarbonLoadOptionConstants.java

```diff
@@ -99,6 +99,15 @@
   public static final String CARBON_OPTIONS_GLOBAL_SORT_PARTITIONS =
       "carbon.options.global.sort.partitions";
 
+  /**
+   * specify serialization null format
```

Comment:
Please describe what is the purpose of this property

Reply:
ok